CMD/Options.cs

```diff
@@ -51,5 +51,8 @@ class Options
 
         [Option("inferStrandedness", Required = false, HelpText = "Infer the strandedness with a sample of the FASTQ files", Default = false)]
         public bool InferStrandSpecificity { get; set; }
+
+        [Option("lncRNAWorkflow", Required =false, HelpText = "Start a lncRNA workflow -skewer-STAR-cufflinks-slncky", Default = false)]
+        public bool lncRNAWorkflow { get; set; }
```

Comment:
I suggest we use a command instead. `CMD.exe -c lncRNA` and check the command option instead. This is what I have started to do above with `"Command: (1) setup, (2) run, (3) vcf2protein, (4) starFusionTest"`